app/views/ads/_ad.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= link_to ad_path(ad.to_param) do %>
```

Comment:
Wrapping all this `div`s with `a` tags works ok?

Reply:
yup, check scheenshots